### PR TITLE
Remove dimensional weight pricing

### PR DESCRIPTION
**Pull request by mbj**
Inverse of: 941512b761d75cf40da524ffe5ea27d0f2ef5a0f

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb This is ready for review.

File: app/models/spree/calculator/shipping/active_shipping/base.rb [hunk 1]
```diff
@@ -178,35 +178,26 @@ def convert_package_to_weights_array(package)
...
         def packages(package)
           units      = Spree::ActiveShipping::Config[:units].to_sym
           packages   = []
           weights    = convert_package_to_weights_array(package)
           max_weight = get_max_weight(package)
-          dimensions = convert_package_to_dimensions_array(package)
 
           if max_weight.to_i <= 0
-            packages << Package.new(weights.sum, dimensions, :units => units)
+            packages << Package.new(weights.sum, [], :units => units)
```
Comment (dkubb) [inline app/models/spree/calculator/shipping/active_shipping/base.rb]: Where is `Package` defined? Would it be possible to remove the second argument to the constructor, and/or define it like this:

```ruby
def initialize(sum, dimensions = [], units:)
  # ...
end
```

Comment (dkubb) [inline app/models/spree/calculator/shipping/active_shipping/base.rb]: This should use 1.9 style Hashes.

File: app/models/spree/calculator/shipping/active_shipping/base.rb [hunk 2]
```diff
@@ -178,35 +178,26 @@ def convert_package_to_weights_array(package)
...
           else
             package_weight = 0
             weights.each do |content_weight|
               if package_weight + content_weight <= max_weight
                 package_weight += content_weight
               else
-                packages << Package.new(package_weight, dimensions, :units => units)
+                packages << Package.new(package_weight, [], :units => units)
```
Comment (dkubb) [inline app/models/spree/calculator/shipping/active_shipping/base.rb]: Instance of https://github.com/MountainRoseHerbs/spree_active_shipping/pull/21/files#r64459522

File: app/models/spree/calculator/shipping/active_shipping/base.rb [hunk 3]
```diff
@@ -178,35 +178,26 @@ def convert_package_to_weights_array(package)
...
                 package_weight = content_weight
               end
             end
-            packages << Package.new(package_weight, dimensions, :units => units) if package_weight > 0
+            packages << Package.new(package_weight, [], :units => units) if package_weight > 0
```
Comment (dkubb) [inline app/models/spree/calculator/shipping/active_shipping/base.rb]: Instance of https://github.com/MountainRoseHerbs/spree_active_shipping/pull/21/files#r64459522

Comment (dkubb) [inline app/models/spree/calculator/shipping/active_shipping/base.rb]: If `package_weight` cannot be less than 0, maybe this should be written as:

```ruby
packages << Package.new(package_weight, [], units: units) unless package_weight.zero?
```

**Comment by dkubb** [comment]
@mbj I finished a pass over this code.

File: app/models/spree/calculator/shipping/active_shipping/base.rb (hunk 1 above)
Comment (mbj) [inline app/models/spree/calculator/shipping/active_shipping/base.rb]: @dkubb Package comes from `active_shipping` itself, I do not want to touch it at this point.

File: app/models/spree/calculator/shipping/active_shipping/base.rb (hunk 3 above)
Comment (mbj) [inline app/models/spree/calculator/shipping/active_shipping/base.rb]: I do not know if `package_weight` could get subzero, I as we can drop splitting here anyway I think we should not touch it for now. And only remove the unused semantics of the dimensional pricing "monkey patch extension API".

**Comment by mbj** [comment]
@dkubb This is ready for another pass.

**Pull request merged**